content/sensu-go/6.1/operations/monitor-sensu/monitor-sensu-with-sensu.md

```diff
@@ -84,6 +84,11 @@ spec:
 
 {{< /language-toggle >}}
 
```

Comment:
Review question: line 45 says the `check-http.rb` command is from Monitoring Plugins, but the listed asset is sensu-plugins-http. I also think the command in Monitoring Plugins uses an underscore rather than a hyphen. Should we change "Monitoring Plugins" on line 45 to Sensu HTTP Plugins instead?

Reply:
Discussed w/Cameron -- change the explanation to Sensu HTTP Plugins and create an issue to standardize on Monitoring Plugins